vuepress/docs/v7.0/tutorials/create/mfe/react.md

```diff
@@ -62,102 +63,99 @@ customElements.define('my-widget', WidgetElement);
+- Cannot be a single word
+- Should follow `kebab-case` naming convention
 :::
 
 ### Display the Custom Element
 
-1. Replace the full contents of `src/index.js` with these two lines: 
+1. Replace the entire contents of `src/index.js` with these two lines: 
 
 ``` js
 import './index.css';
 import './WidgetElement';
 ```
 
-2. Edit `public/index.html` and replace `<div id="root"></div>` with this:
+2. In `public/index.html`, replace `<div id="root"></div>` with this:
 
 ``` html
     <my-widget />
 ```
 
-3. Your browser window should automatically redisplay the React App.
+3. Observe your browser automatically redisplay the React app
 
 ::: tip Congratulations!
-You’re now displaying a React App using a custom element.
+You’re now using a custom element to display a React app.
 :::
 
-Now we'll take this custom element and put it into Entando as a micro frontend.
-
 ## Display the Micro Frontend in Entando
 
 ### Build the React App
 
-First we'll need to perform a production build of the React App
+To deploy the custom element into Entando as a micro frontend, you must perform a production build of the React app.
 
-1. Create an `.env.production` file in the `my-widget` directory. It should have one line:
+1. In the `my-widget` directory, create an `.env.production` file that consists of one line:
 ``` text
 PUBLIC_URL=/entando-de-app/cmsresources/my-widget
 ```
 
 ::: warning Notes
-- `/entando-de-app/cmsresources/` is the Resource URL for your Entando application. This matches nearly all development environments. Consult your Entando admin or use the following fragment to discover the Resource URL.
+- `/entando-de-app/cmsresources/` is the Resource URL for your Entando Application, which matches nearly all development environments. Consult your Entando admin or use the following fragment to discover the Resource URL.
 ``` ftl 
   <#assign wp=JspTaglibs[ "/aps-core"]>
   <@wp.resourceURL />
 ``` 
-- `/my-widget` is the public folder you'll use to host the JavaScript and CSS files for the React App
+- `/my-widget` is the public folder that hosts the JavaScript and CSS files for the React app
 :::
 
 
-2. Go to the `my-widget` directory and build the app
-
+2. Build the app:
 ``` bash
 npm run build
 ```
 
-Now you are ready to finish setting up and using the micro frontend in Entando.
-
 ### Upload the React files
 
-1. In the App Builder, go to 'Administration' → 'File browser' → 'public'
+To set up the micro frontend to use in Entando:
+
+1. In your App Builder, go to `Administration` → `File browser` → `public`
 
-2. Click `Create folder` and name it 'my-widget'. This needs to match the .env.production path from above.
+2. Click `Create folder` and name it "my-widget" to match the `.env.production` path above
 
 3. Click `Save`
 
 4. Click `my-widget`
 
-5. Create the same folder structure as your generated build directory
+5. Create a folder structure similar to your generated build directory:
```

Comment:
similar threw me off a little, because it made me wonder if I could change the names a littl

Reply:
you can! strictly speaking the structure isn't identical